### PR TITLE
feat: Mesh Manager API specs

### PR DESCRIPTION
**Pull request by cloudjumpercat**
### Description

Currently (for beta), we just had a link to beta specs. I needed to add a new API spec in the new format for Mesh Manager. 
 

DOCU-3481
### Testing instructions

Preview link: 

### Checklist 

- [x] Review label added 
- [x] PR pointed to correct branch (`main` for immediate publishing, or a release branch: e.g. `release/gateway-3.2`, `release/deck-1.17`)

### CODE REVIEW

**Comment by cloudjumpercat** [comment]
Build is currently failing because I have a placeholder for the ID in the spec page right now.

**Comment by Guaris** [comment]
If you make the page in konnect you can use the real ID, you can also use the real ID for the version if you make that too you don't need to upload a spec. You still have to build the konnect_oas_data.json schema before this can be used.

File: app/_data/konnect_oas_data.json [hunk 1]
```diff
@@ -170,6 +170,27 @@
       }
     ]
   },
+  {
+    "id": "f3675942-dbc0-4298-bec2-cbcfdcc10fab",
+    "title": "Konnect Mesh Manager",
+    "latestVersion": {
+      "name": "v1",
+      "id": "0426ee88-bf76-45e3-9f89-fc03bea0c3a5"
```
Comment (cloudjumpercat) [inline app/_data/konnect_oas_data.json:178]: This wasn't in the UI (that I could see), so I used the API product spec to find it. Could someone double check that this is correct?

Comment (lena-larionova) [reply app/_data/konnect_oas_data.json:178]: It's right. You can grab it from the URL:

```
https://cloud.konghq.com/us/api-products/f3675942-dbc0-4298-bec2-cbcfdcc10fab/versions/0426ee88-bf76-45e3-9f89-fc03bea0c3a5
```
Where the last ID after `versions/` is the one you want.

Comment (acgoldsmith) [reply app/_data/konnect_oas_data.json:178]: URL is the only place I've seen version specified, you got it right

**Review by lena-larionova — APPROVED** [review]

File: app/_data/konnect_oas_data.json [hunk 2]
```diff
@@ -170,6 +170,27 @@
...
+    "versionCount": 1,
+    "versions": [
+      {
+        "id": "0426ee88-bf76-45e3-9f89-fc03bea0c3a5",
+        "created_at": "2023-09-25T12:01:47.449Z",
+        "updated_at": "2023-09-25T12:02:03.266Z",
+        "name": "v1",
+        "deprecated": false,
+        "registration_configs": []
+      },
```
Comment (lena-larionova) [inline app/_data/konnect_oas_data.json:191]:
```suggestion
      }
```

**Pull request merged**